src/data/finsupp.lean

```diff
@@ -1324,6 +1325,23 @@ end curry_uncurry
 
 section
 
+instance comap_has_scalar [group γ] [mul_action γ α] [add_comm_monoid β] : has_scalar γ (α →₀ β) :=
+{ smul := λ g f, f.comap_domain (λ a, g⁻¹ • a)
+  (λ a a' m m' h, by simpa [←mul_smul] using (congr_arg (λ a, g • a) h)) }
```

Comment:
I'm not sure that this should be a global instance.

Reply:
I agree, I'll remove this.